packages/evm/test/precompiles/01-ecrecover.spec.ts

```diff
@@ -3,7 +3,8 @@ import { bytesToHex, bytesToUnprefixedHex, hexToBytes, utf8ToBytes } from '@ethe
 import { keccak256 } from 'ethereum-cryptography/keccak.js'
 import { assert, describe, it } from 'vitest'
 
-import { EVM, getActivePrecompiles } from '../../src/index.js'
+import { EVM } from '../../src/evm.js'
+import { getActivePrecompiles } from '../../src/precompiles/index.js'
```

Comment:
All these import changes for `EVM` and `getActivePrecompiles` and some others are not so nice, also can't totally see why this is necessary or what this is doing. Is this really needed? We should really try to avoid. Not seeing why especially here the normal re-export we have everywhere from `index.js` should not also work?

Reply:
@ScottyPoi, is this because vitest can't find the imports if you import from `../../index.js`?  I've seen that sort of thing before where a test that's in a `test/[subdirectory]` can't find imports that are in `../../src/[subdirectory]/someImport.js`

Reply:
Ok, I was never able to resolve that module not found thing.  It's something in vitest and I always had to specify complete relative paths to modules that were in subdirectories.  Vitest has some trouble resolving re-exported modules where the importing module and the original export are in different subdirectories at different layers from the root.

I asked on the Vitest discord once and just got a shrug.

Reply:
I think I solved some of this by making changes to  `index.ts` files in `evm` itself.

The solution seems to be to make sure there is a consistency with `import` vs `import type` and `export` vs `export type` for each thing exported from an `index.ts` file.